client/components/mma/accountoverview/InAppPurchaseCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import { from, headline, palette, space } from '@guardian/source-foundations';
+import type { AppSubscription } from '../../../../shared/mpapiResponse';
+import { InfoSummary } from '../paymentUpdate/Summary';
+import { Card } from '../shared/Card';
+
+const productTitleCss = css`
```

Comment:
I'm adding a cancelled product card component too so we'll probably want to move some of the product card styles to someone shared rather than duplicating them, but it'll be simpler to do that once we've merged both of them and can see what we have.